Makefile.toml

```diff
@@ -1401,9 +1401,20 @@ script = [
     '''
 ]
 
-# This task will clear all tests and resources from the testsys cluster.
+# This task will clear all passed tests from the testsys cluster.
 [tasks.clean-test]
 dependencies = ["test-tools"]
+script = [
```

Comment:
I think we should keep the previous behavior of `clean-test` and introduce a new task `clean-passed-test` instead. This way we keep backwards compatibility and I think it's probably more clear that way as well.

Reply:
I think the new `cargo make clean-test --passed`  makes sense. Thanks!